examples/dataset/object_detection_3d/object_detection_3d/utils.py

```diff
@@ -181,3 +184,65 @@ def load_data(df_raw: pd.DataFrame) -> pd.DataFrame:
         )
 
     return pd.DataFrame(records)
+
+
+def _compute_thresholded_metrics(
```

Comment:
What's the reason for re-implementing these evaluation functions here? Could we leverage `multiclass_datapoint_metrics` from `kolena/_experimental/object_detection/dataset.py` instead?